Civi/Payment/System.php

```diff
@@ -53,18 +53,20 @@ public function getByProcessor($processor, $force = FALSE) {
       }
       else {
         $paymentClass = 'CRM_Core_' . $processor['class_name'];
-        if (empty($paymentClass)) {
+        if (empty($processor['class_name'])) {
           throw new \CRM_Core_Exception('no class provided');
         }
-        require_once str_replace('_', DIRECTORY_SEPARATOR, $paymentClass) . '.php';
+        if (class_exists($paymentClass)) {
+          require_once str_replace('_', DIRECTORY_SEPARATOR, $paymentClass) . '.php';
```

Comment:
I wonder if we still need this?

Reply:
actually by definition we don't - ie. class_exists will fail if not already included